lib/agent0/agent0/hyperdrive/crash_report/crash_report.py

```diff
@@ -49,11 +49,13 @@ def setup_hyperdrive_crash_report_logging(log_format_string: str | None = None)
 
 
 # pylint: disable=too-many-statements
+# pylint: disable=too-many-branches
```

Comment:
I propose putting the limits on these to double their current values instead of playing this game where we disable all the time

Reply:
I'm all for that